tests/configlet/util/common.py

```diff
@@ -235,8 +238,13 @@ def get_dump(duthost, db_name, db_info, dir_name, data_dir):
     db_write = {}
     for k in db_read:
         # Transient keys start with "_"; Hence skipped
-        if ((not k.startswith("_")) and (not match_key(k, keys_skip_cmp))):
-            db_write[k] = {} if match_key(k, keys_skip_val) else db_read[k]
+        if (not k.startswith("_")) and (not match_key(k, keys_skip_cmp)):
+            value = db_read[k].get("value", {})  # Get the value or empty dictionary if
+
+            for skip_val in keys_skip_val:
```

Comment:
This logic will pop items during the iteration of a dict.
Have you tried on a real case that a key is really poped?
I believe it will raise an exception complaining this.

A sample code:
https://github.com/sonic-net/sonic-mgmt/blob/a0c0c2bef9fa431a73c82f004d9575cb279f9b3e/tests/common/fixtures/duthost_utils.py#L690C11-L690C14

Reply:
Hi @yejianquan , there is some trick in here.
If we loop value, then pop. It will raise exception.
```
//result in failure
for v in value:
    if match_key(v, keys_skip_val):
        value.pop(v)
```
I choose to loop `keys_skip_val`, which is a fix length dict. Then I pop the element in value. There will be no exception.

I did the E2E test on DUT, and make sure it really popped.